commit 7bbb46c1e2d22e7a144bcca18f6e0ca1e0eaf8e2
Author: Jean-Baptiste Kempf <jb@videolan.org>
Date:   Thu Mar 5 12:11:43 2009 +0100

    Remove insult in the fortunes...
    
    This doesn't mean I don't share the opinion, but let's try to ease the work for packagers that get bugs on that part.
    And of course VLC + debian-multimedia in debian is still completly broken...

diff --git a/doc/fortunes.txt b/doc/fortunes.txt
index 9406996..2ee386f 100644
--- a/doc/fortunes.txt
+++ b/doc/fortunes.txt
@@ -76,20 +76,6 @@
 
   -- #videolan
 %
-<sam> CHRISTIAN
-<sam> MARILLAT
-<sam> IS
-<sam> A
-<sam> COMPLETE
-<sam>  ____ ___ ____ _  ___   _ _____    _    ____    _ 
-<sam> |  _ \_ _/ ___| |/ / | | | ____|  / \  |  _ \  | |
-<sam> | | | | | |   | ' /| |_| |  _|   / _ \ | | | | | |
-<sam> | |_| | | |___| . \|  _  | |___ / ___ \| |_| | |_|
-<sam> |____/___\____|_|\_\_| |_|_____/_/   \_\____/  (_)
-<sam> 
-
-  -- #videolan
-%
 <bozo> you really shouldn't have touched a computer when you were a little boy
 <stef> _it_ came to me
 
